Jellyfin.Api/Controllers/DisplayPreferencesController.cs

```diff
@@ -126,6 +126,7 @@ public DisplayPreferencesController(IDisplayPreferencesManager displayPreference
             HomeSectionType.LiveTv,
             HomeSectionType.NextUp,
             HomeSectionType.LatestMedia,
+            HomeSectionType.LatestRelease,
             HomeSectionType.None,
```

Comment:
There is a limit of 6 home sections so adding this one doesn't make any sense, in fact the latest ".None" value doesn't make sense either.

Reply:
It seems that the limit is not 6, but 7 in the current web version.
![image](https://github.com/jellyfin/jellyfin/assets/12448284/d8c3fbba-a522-42fe-b3c9-d5d646f2af21)

I agree that setting 7 or 8 different ones by default doesn't need to be. Instead the UI should offer a option to add a new section on demand.

Reply:
I removed the addition of the new home section type as it doesn't make a lot of sense if the UI needs to be changed everytime as previously stated.